### ISSUE TITLE
Text field does not collapse to represent used space.

### ISSUE THREAD

**Issue by xpkoala** [Crypton/statusapp#166]
The 'post box' on the Timeline does not display the correct use of space for entries in real time. One the last picture there is NO information past the first line. All the subsequent text and new line entries had been removed.

iphone 5s 9.0.1

>-Select 'new post' button
>-Enter random information or new lines to add length to the post box.
>-Remove all text.

![img_1060](https://cloud.githubusercontent.com/assets/11012743/10412030/d152a768-6f3e-11e5-916f-f25fd2954ae9.PNG)
![img_1062](https://cloud.githubusercontent.com/assets/11012743/10412031/d621696e-6f3e-11e5-9de1-ec72c724356e.PNG)
![img_1063](https://cloud.githubusercontent.com/assets/11012743/10412032/d8f1edbc-6f3e-11e5-97ee-352ec2bb1352.PNG)

**Comment by daviddahl** [comment]
Yeah, this is I think a corner case for now. We need to add another function to fire on the keyup event where we check for the number of rows of text and adjust the height accordingly.
